src/shared/models/licensemodel.cpp

```diff
@@ -25,6 +26,12 @@ LicenseModel* LicenseModel::instance() {
 
 LicenseModel::LicenseModel(QObject* parent) : QAbstractListModel(parent) {
   MZ_COUNT_CTOR(LicenseModel);
+
+  connect(ResourceLoader::instance(), &ResourceLoader::cacheFlushNeeded, this,
+          [this]() {
+            m_licenses.clear();
+            initialize();
+          });
```

Comment:
It feels fragile that all consumers of the ResourceLoader singleton need to intrument this signal. Any way we can make this not necessary or obviously enforced?

Reply:
nope. But the good thing is that, if we don't do it, the resource cannot be replaced. Not a big deal.